users/src/commonMain/kotlin/majestic/users/profile/roles/index.kt

```diff
@@ -0,0 +1,274 @@
+package majestic.users.profile.roles
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import captain.Navigator
+import composex.screen.orientation.Landscape
+import composex.screen.orientation.Portrait
+import composex.screen.orientation.ScreenOrientation
+import majestic.ColorPair
+import majestic.ExpandDirection
+import majestic.FloatingActionButton
+import majestic.floatActionButton
+import majestic.tooling.onClick
+import majestic.users.labels.roles.RolesLabels
+import majestic.users.labels.settings.LanguageController
+import majestic.users.labels.settings.observeUsersLabels
+import majestic.users.profile.roles.assignRoleModalContent.AssignRoleModalContent
+import majestic.users.profile.roles.campus.Campus
+import majestic.users.profile.roles.campus.CampusMenuAction
+import majestic.users.tools.ProfilePortraitHeader
+import majestic.users.tools.buttons.ButtonAnimate
+import majestic.users.tools.buttons.FlatButton
+import majestic.users.tools.dialogs.Modal
+import tz.co.asoft.majestic_users.generated.resources.Res
+import tz.co.asoft.majestic_users.generated.resources.ic_add
+
+@Composable
+private fun RolesHeader(
```

Comment:
directly inline the body of this composable at the call site. Because on portrait it actually doesn't render anything and on landscape all it does is just render a Text. So, just do the check directly at the call site and render the Text there